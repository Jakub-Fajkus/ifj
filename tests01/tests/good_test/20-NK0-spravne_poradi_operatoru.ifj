/* test20 tak chvalne :D  */
class Main
{
	static void run()
	{
		int a; int b; int c; int vysl; int vysl2;
		a = 6;
		b=5;
		c=2;
		
		vysl=a+b+c*c/4+8-b-a*c;		//vysledek je 3
		vysl2=(a+b+((c*c)/4)+8-b-(a*c));	//pro kontrolu
		
		ifj16.print("Vysledek bez zavorek: " + vysl + "\n");
		ifj16.print("Vysledek se zavorkama: " + vysl2 + "\n");
		

	}
	
}