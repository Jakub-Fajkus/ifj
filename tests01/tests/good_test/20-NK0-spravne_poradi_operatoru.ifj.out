Vysledek bez zavorek: 3
Vysledek se zavorkama: 3
